ext/libstrawberry-common/backtrace_inc.h.in

```diff
@@ -0,0 +1,25 @@
+/*
+ * Strawberry Music Player
+ * Copyright 2013, Jonas Kvinge <jonas@strawbs.net>
+ *
+ * Strawberry is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Strawberry is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Strawberry.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#cmakedefine01 Backtrace_FOUND
```

Comment:
Can this be moved to src/config.h.in ?

Reply:
Why? It's only ever used in one place, why pollute global includes with it?